Python/math/euler_totient_function.py

```diff
@@ -0,0 +1,21 @@
+# python program for Euler's Totient Function 
+import math
+def euler_totient(q):
+    ans=q   #defining the answer initially to n
```

Comment:
The function give wrong answer for various sample cases. ex- 4, 6, 9 etc. Please check the algorithm used once again.